### DISCUSSION TITLE
Disabling the photo album option in PWA Elements camera

### DISCUSSION THREAD

**Discussion by niklasjk** [ionic-team/capacitor#3902] [Q&A]
Hello,

Is it possible to somehow disable the photo album option in the PWA Elements camera? I have chosen CameraSource.Camera as the source for the getPhoto method but that only seems to affect the camera on Android and iOS. I noticed that the button is not visible in the example image on the PWA Elements page here https://capacitorjs.com/docs/web/pwa-elements.

**Comment by tlloydukdev** [comment]
Also looking for a solution to this 👍

**Reply by tlloydukdev** [reply]
In the end I used `patch-package` to customise the @iconic/pwa-elements module so it didn't render the Pick Photo icon. Would be great if this was added as a config option in the future.

**Reply by diadal** [reply]
can you share this solution @tlloydukdev

**Reply by tlloydukdev** [reply]
in the end I didn't use PWA Elements and opted for a standard html file input tag with capture="camera" attribute to remove the album option. This is because PWA Elements didn't work well with Safari/iOS
